src/backend/distributed/planner/multi_join_order.c

```diff
@@ -1157,12 +1198,20 @@ ApplicableJoinClauses(List *leftTableIdList, uint32 rightTableId, List *joinClau
 }
 
 
+/* Returns the left expression in the given join clause. */
+Node *
+LeftHandExpr(OpExpr *joinClause)
+{
+	List *argumentList = joinClause->args;
+	return (Node *) linitial(argumentList);
+}
+
+
 /* Returns the left column in the given join clause. */
 Var *
 LeftColumn(OpExpr *joinClause)
```

Comment:
There's only two usages of this function left (in `multi_logical_planner.c`). Can these be changed to the new `orNULL` variants as well?